ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_node_type.schema.yml

```
@@ -671,6 +671,12 @@ $defs:
               table: node-type-l2-mlag-configuration
             description: MLAG Domain ID. If not set the node group name (Set with "group" key) will be used.
             type: str
+          mlag_ptp:
+            documentation_options:
+              table: node-type-l2-mlag-configuration
+            description: Determine default behavior for configuring PTP on the MLAG peer link port-channel when PTP is enabled. If not explicitly set, PTP will not be enabled on the MLAG peer link port-channel.
```

We are not changing the default behavior as such. This can be set on node, node_group or defaults.

```suggestion
            description: Configure PTP on the MLAG peer-link port-channel when PTP is enabled. If not explicitly set, PTP will not be configured on the MLAG peer-link port-channel.
```